airbyte-integrations/connectors/source-stripe/source_stripe/streams.py

```diff
@@ -835,3 +838,17 @@ def get_error_handler(self) -> Optional[ErrorHandler]:
         return ParentIncrementalStripeSubStreamErrorHandler(
             logger=self.logger, error_mapping=PARENT_INCREMENTAL_STRIPE_SUB_STREAM_ERROR_MAPPING
         )
+
+
+class UpdatedCursorIncrementalStripeSubStream(UpdatedCursorIncrementalStripeStream, HttpSubStream):
```

Comment:
This class seems oddly familiar to `Persons`. Is there a difference there? Should we consolidate the two of them?

Reply:
agreed

Reply:
ref `persons` to use `UpdatedCursorIncrementalStripeSubStream`